docs/docs/guides/04_transactions/gas-and-fees.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 3
+sidebar_label: 'Gas and Priority Fees'
+---
+
+# Gas and Priority Fees
+
+To prevent spam and reward node operators, Ethereum uses a mechanism called ["gas"](https://ethereum.org/en/gas/), which is a cost that is charged to execute a transaction. Gas costs are associated with all transactions that require computation or update state (e.g. transferring tokens from one account to another). Gas costs are not applied to requests that only involve reading state (e.g. querying the balance of an account). The amount of gas required to pay for a transaction is calculated based on the actual computational actions that are required to execute that transaction. For instance, a simple transaction to transfer ETH from one account to another will require less gas than invoking a complicated smart contract function that involves lots of computation and storage. The cost of gas varies based on network usage. Gas costs more during period of high network activity. The cost per unit of gas is known as the "base fee".
+
+In addition to the calculated gas cost, an Ethereum transaction can specify an optional priority fee, which is an additional fee that is paid directly to the operator of the node that executes the transaction. Priority fees are intended to incentive node operators to execute transactions. A priority fee is specified as a value in addition to the base fee, which means that the total cost of the priority fee is always a factor of the amount of gas required to execute a transaction.
+
+With the above in mind, the total cost of fees associated with a transaction are calculated as: _units of gas used \* (base fee + priority fee)_.
```

Comment:
```suggestion
With the above in mind, the total cost of fees associated with a transaction are calculated as:

Total cost of fees =  _units of gas used \* (base fee + priority fee)_.
```
suggestion - i think writing it out on a new line and as a full math equation can make it easier for the user to read and stands out a bit more